core/src/main/java/com/linecorp/armeria/common/stream/StreamMessage.java

```diff
@@ -1136,4 +1136,12 @@ default InputStream toInputStream(Function<? super T, ? extends HttpData> httpDa
     default StreamMessage<T> endWith(Function<@Nullable Throwable, ? extends @Nullable T> finalizer) {
         return new SurroundingPublisher<>(null, this, finalizer);
     }
+
+    /**
+     * Run subscribe, onSubscribe and request on the specified {@link EventExecutor}
```

Comment:
Question: What does the request mean here?
Would you also be able to describe it specifically?
e.g. `{@link #subscribe(Subscriber, EventExecutor, SubscriptionOption...)} and methods of the {@link Subscriber}`

Reply:
I just copied over the javadocs from reactor, but I agree the javadoc could be improved.
https://projectreactor.io/docs/core/release/api/reactor/core/publisher/Flux.html#subscribeOn-reactor.core.scheduler.Scheduler-

Updated and also added an example